### ISSUE TITLE
NewWorkItem.cs - bug

### ISSUE THREAD

**Issue by sztaylorakgov** [igoravl/TfsCmdlets#191]
@igoravl In 2.4.1 and later, I started having problems with New-TfsWorkItem related to area path.  Upgraded to 2.5.0 but problem was still there.  Reviewed commits and I think I might see the problem in NewWorkItem.cs (2.5.0+2840.2):

```        
        /// <summary>
        /// Specifies the area path of the work item.
        /// </summary>
        [Parameter]
        [WorkItemField("System.IterationPath", FieldType.TreePath)]
        public string AreaPath { get; set; }

        /// <summary>
        /// Specifies the iteration path of the work item.
        /// </summary>
        [Parameter]
        [WorkItemField("System.AreaPath", FieldType.TreePath)]
        public string IterationPath { get; set; }
```

Unless I'm misreading the code, it looks like the WorkItemField declarations are reversed.  Looking back at 2.4.1+2827.1, I see a similar issue that would explain the problem I originally saw in 2.4.1.

I'm not setup to test and contribute a fix, but I did downgrade to 2.4.0 and found the problem with New-TfsWorkItem goes away.  So, hopefully, this is helpful enough info to make the fix easy.

**Comment by igoravl** [comment]
Thanks for the catch. @sztaylorakgov - and apologies for the inconvenience.

I just issued a fix - please update to 2.5.1.

**Comment by sztaylorakgov** [comment]
Thanks, @igoravl!  Upgraded to 2.5.1 and tested New-TfsWorkItem including params for Area and Iteration Path values.  Works!
